### PR TITLE
docs: Revamp the README

### PR DESCRIPTION
**Pull request by BarbzYHOOL**
Viewable here https://github.com/azerothcore/azerothcore-wotlk/blob/bb7c23dbaf59360b1af0366447061384909a1f9c/README.md

##### CHANGES PROPOSED:
- Useful links
- Better presentation

##### Target branch(es):
- [x] Master

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,66 +3,171 @@
...
 [![Master Build Status](https://travis-ci.org/azerothcore/azerothcore-wotlk.svg)](https://travis-ci.org/azerothcore/azerothcore-wotlk)
 <a href="https://scan.coverity.com/projects/azerothcore-azerothcore-wotlk">
-  <img alt="Coverity Scan Build Status"
-       src="https://scan.coverity.com/projects/13814/badge.svg"/>
+  <img alt="Coverity Scan Build Status"  src="https://scan.coverity.com/projects/13814/badge.svg"/>
 </a>
 [![Bountysource](https://www.bountysource.com/badge/tracker?tracker_id=40032087)](https://www.bountysource.com/teams/azerothcore/bounties)
 [![StackOverflow](http://img.shields.io/badge/stackoverflow-azerothcore-blue.svg)]( https://stackoverflow.com/questions/tagged/azerothcore?sort=newest)
-[![Discord](https://img.shields.io/discord/217589275766685707.svg)](https://discord.gg/gXNKCG5)
+[![Discord](https://img.shields.io/discord/217589275766685707.svg)][discord]
```
Comment (FrancescoBorzi) [inline README.md]: this is wrong, look: https://github.com/azerothcore/azerothcore-wotlk/blob/c477d5e3ed03684504f16194713401251762e32b/README.md

Comment (BarbzYHOOL) [reply README.md]: yeah all links are bugged -__- sorry i didn't expect markdown to not work correctly on github, i need to change it all

File: README.md [hunk 2]
```diff
@@ -3,66 +3,171 @@
...
+1. A lot of modules to choose from
+1. Better configuration files system
+1. Compatibility with other emulators
+1. Friendly and helpful community
 
-We also believe in [software modularity](https://en.wikipedia.org/wiki/Modular_programming), because the more the software is modular, and the more it will be easy to be maintained, understood, developed, etc... So we organized the structure of AzerothCore as modular as possible, giving the possibility to [create new modules](http://www.azerothcore.org/wiki/Create-a-Module) to add custom features (instead of patching the core).
+#### Stability
 
-## Learning project
+As players and administrators ourselves, we take great care into the stability of our core. Other projects focus on the development side of things and tend to forget that users want stability above everything else.
```
Comment (FrancescoBorzi) [inline README.md]: we might add something about testing here, e.g. "No one is pushing commits directly to the core. All changes are **reviewed** and **tested** before they get to the `master` branch."

File: README.md [hunk 3]
```diff
@@ -3,66 +3,171 @@
...
+#### Modules
 
-Installation instructions are available [here](http://www.azerothcore.org/wiki/Installation).
+Modules are essential to AzerothCore's success. Modules allows users to plug them in and out easily, and do not require to modify the core files. It also means users can keep pulling the git changes from the main repository and only develop their own modules.
+
+We have a lot of modules already made, some of them are very important and will ease your work:
+
+* [Transmogrification](https://github.com/azerothcore/mod-transmog)
+* [Eluna (lua engine) support](https://github.com/azerothcore/mod-eluna-lua-engine/ "Creator of Eluna is part of our core team")
+* [Almost full list available here][catalogue]
```
Comment (FrancescoBorzi) [inline README.md]: missing catalogue link

File: README.md [hunk 4]
```diff
@@ -3,66 +3,171 @@
...
+
+#### Configuration files
+
+Our configuration file system allows the user to use a tiny configuration file for better readability and maintenance.
+
+#### Compatibility with other emulators
+
+Not very far from its ancestor TrinityCore, most scripts can be adapted quite easily. For Mangos compatibility, it might require more knowledge but it shares a common base.
```
Comment (FrancescoBorzi) [inline README.md]: please replace `Mangos` with `MaNGOS`

File: README.md [hunk 5]
```diff
@@ -3,66 +3,171 @@
...
 
-## Learning project
+As players and administrators ourselves, we take great care into the stability of our core. Other projects focus on the development side of things and tend to forget that users want stability above everything else.
 
-AzerothCore is a learning project.
-If you want to learn how to use C++, SQL, git, etc... you came to the right place! We will gladly help you: **programming is our passion**.
+#### Authenticity
 
-If you want to contribute, in [our wiki](http://www.azerothcore.org/wiki) you fill find a lot of resources that will guide you. Also you can poke us in [our discord chat server](https://discord.gg/gkt4y2x).
+Fixing and implementing missing blizzlike content is one of our priority, and we can boast to offer the most content-complete open source emulator.
```
Comment (FrancescoBorzi) [inline README.md]: "one of our priorities" ?

File: README.md [hunk 6]
```diff
@@ -3,66 +3,171 @@
...
+
+Here are the main points we focus on:
+
+* Stability
+* Ease of use / Practicability
+* Playability (ingame content)
+* Customization
+* Community driven software (check our discord)
+
+We also welcome new users (even non-english speaking users!) and help them learn / improve their skills (C++, SQL, Git, software collaboration, tutoring (wiki) etc...).
```
Comment (FrancescoBorzi) [inline README.md]: nested parentheses

File: README.md [hunk 7]
```diff
@@ -3,66 +3,171 @@
...
+By talking about us on different platforms or to people who would like to get involved.
 
-For help requests, it is recommended to ask your question on [StackOverflow](https://stackoverflow.com/questions/tagged/azerothcore) and link it in [our chat](https://discordapp.com/channels/217589275766685707/284406375495368704).
 
-## Docker
+## CONTRIBUTING
+
+AzerothCore is a learning project, and there are lots of different ways to contribute to the project:
+
+* By testing our fixes (we can teach you how to correctly use Git to help us but that will also help you out tremendously)
```
Comment (FrancescoBorzi) [inline README.md]: add link to http://www.azerothcore.org/wiki/How-to-test-a-PR

File: README.md [hunk 8]
```diff
@@ -3,66 +3,171 @@
...
+* By developing directly to the core or to the modules
+* By [creating new modules](http://www.azerothcore.org/wiki/Create-a-Module)
+* By improving our [wiki][wiki]
```
Comment (FrancescoBorzi) [inline README.md]: missing wiki link

File: README.md [hunk 9]
```diff
@@ -3,66 +3,171 @@
...
+* By providing direct support to our community (on discord, stackoverflow or specialized forums)
+* By making extra content (video tutorial for example)
+* By putting bounties on issues
+
+
+If you want to contribute to the project, you will find a lot of resources that will guide you in [our wiki][wiki].
+
+Feel free to join us on [our discord chat server][discord] where we taught a lot of new people how to get started and who are now important contributors!
```
Comment (FrancescoBorzi) [inline README.md]: missing discord link

File: README.md [hunk 10]
```diff
@@ -3,66 +3,171 @@
...
+* By testing our fixes (we can teach you how to correctly use Git to help us but that will also help you out tremendously)
+* By developing directly to the core or to the modules
+* By [creating new modules](http://www.azerothcore.org/wiki/Create-a-Module)
+* By improving our [wiki][wiki]
+* By providing direct support to our community (on discord, stackoverflow or specialized forums)
+* By making extra content (video tutorial for example)
+* By putting bounties on issues
+
+
+If you want to contribute to the project, you will find a lot of resources that will guide you in [our wiki][wiki].
```
Comment (FrancescoBorzi) [inline README.md]: same here

File: README.md [hunk 11]
```diff
@@ -3,66 +3,171 @@
...
+
+Feel free to join us on [our discord chat server][discord] where we taught a lot of new people how to get started and who are now important contributors!
+
+<!-- TO UNCOMMENT LATER -->
+<!-- As we put a big emphasis on community, there are also special rewards for contributors such as reputation ranks (displayed in our discord and our website), reputation badges (to display on your project/portfolio), premium software licenses, private modules access, private tools access and a lot of small private repositories access. -->
+
```
Comment (FrancescoBorzi) [inline README.md]: ?

File: README.md [hunk 12]
```diff
@@ -3,66 +3,171 @@
...
+**DOCKER**
 
 Dockerization of AzerothCore is fully supported. More details about Docker and the setup instructions are available [here](http://www.azerothcore.org/wiki/Install-with-Docker).
 
-## Authors & Contributors
 
-This project is brought to you thanks to:
+## SUPPORT
+
+Our self-made [wiki][wiki] probably has a lot of answers for you.
```
Comment (FrancescoBorzi) [inline README.md]: missing wiki link

File: README.md [hunk 13]
```diff
@@ -3,66 +3,171 @@
...
 
-- The new AzerothCore source components are released under the [GNU AGPL v3](https://github.com/azerothcore/azerothcore-wotlk/blob/master/LICENSE-AGPL3)
-- The old sources based on MaNGOS/TrinityCore are released under the [GNU GPL v2](https://github.com/azerothcore/azerothcore-wotlk/blob/master/LICENSE-GPL2)
-
-## Links
+## IMPORTANT LINKS
 
-- [Website](https://azerothcore.github.io/)
+- [Website](http://www.azerothcore.org/)
+- [AzerothCore catalogue][catalogue] (modules, tools etc...)
```
Comment (FrancescoBorzi) [inline README.md]: missing catalogue link

File: README.md [hunk 14]
```diff
@@ -3,66 +3,171 @@
...
+## LICENSE
+
+- The new AzerothCore source components are released under the [GNU AGPL v3](https://github.com/azerothcore/azerothcore-wotlk/blob/master/LICENSE-AGPL3)
+- The old sources based on MaNGOS/TrinityCore are released under the [GNU GPL v2](https://github.com/azerothcore/azerothcore-wotlk/blob/master/LICENSE-GPL2)
+
+
+<!-- REPEATED LINKS LIST -->
+[catalogue]: http://www.azerothcore.org/modules-catalogue/  "Modules, tools, and other things for AzerothCore" {target=_blank}
+[wiki]: http://www.azerothcore.org/wiki "Easy to use and developed by AzerothCore founder" {target=_blank}
+[discord]: https://discord.gg/gkt4y2x "Our community hub" {target=_blank}
```
Comment (FrancescoBorzi) [inline README.md]: remove this crap, it doesn't work lol

**Comment by FrancescoBorzi** [comment]
@BarbzYHOOL good job!

File: README.md (hunk 2 above)
Comment (BarbzYHOOL) [reply README.md]: ok doing it

File: README.md (hunk 11 above)
Comment (BarbzYHOOL) [reply README.md]: ah hmm i put this for later when we have a special page dedicated to this, lol

File: README.md (hunk 14 above)
Comment (BarbzYHOOL) [reply README.md]: works perfectly fine everywhere else :(

**Comment by BarbzYHOOL** [comment]
@FrancescoBorzi fixed it all, and i stopped the build on travis, coz unneeded, lets see what happens

**Comment by BarbzYHOOL** [comment]
last force-push: I tried to use `[skip travis]` in my commit msg, for curiosity

**Pull request merged**